Translation corrections (#393)
07th-mod/higurashi-console-arcs: Update/omot_009_2.txt

	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#5ec69a>詩音</color>", NULL, "<color=#5ec69a>Shion</color>", NULL, Line_ContinueAfterTyping); }
	ModPlayVoiceLS(4, 6, "ps3/s14/06/210600310", 256, TRUE);
	OutputLine(NULL, "「…事が大きいだけにそれがはっきりしないと、悪いですけど言う通りには従えませんよ」",

> 			NULL, "\"...Sorry, but I can't do what you say unless you make it clear why this is a big deal.\"", Line_Normal);

Reviewer: ```suggestion
			NULL, "\"...Sorry, but I can't do as you ask unless you make it clear why this is a big deal.\"", Line_Normal);
```
